### DISCUSSION TITLE
Custom timeframe in timeline (Gantt Chart)

### DISCUSSION THREAD

**Discussion by FFaranna** [Leantime/leantime#1956] [Ideas]
it would be very helpful to define a custom timeframe in the timeline (Gantt Chart)

![image](https://github.com/Leantime/leantime/assets/7949710/c1d3470f-4bc5-4ed1-b331-809d3d97f3ae)

**Comment by marcelfolaron** [comment]
What custom Timeframe do you need?
Or is it more about setting start and end date and seeing all tasks within
that timeframe?

**Reply by FFaranna** [reply]
start and end date and seeing all tasks within that timeframe would be what I'm looking for.
cheers!

**Reply by root9191** [reply]
is there no timeline view anymore?

**Reply by marcelfolaron** [reply]
The timeline view is under ToDos since it now contains Milestones and Tasks.

**Reply by root9191** [reply]
OK i found it but the tasks do not open. I just get a blank page..

**Comment by marcelfolaron** [comment]
What version are you on?

**Reply by root9191** [reply]
2.4

**Reply by root9191** [reply]
it happens when i wanna click to show tasks on that milestone

**Comment by marcelfolaron** [comment]
Oh I see, the link is wrong. That is a bug.

**Reply by root9191** [reply]
OK, also i cannot save the filter to show only milestones. After new login it also shows the tasks again.
